src/CardinalityEstimator.md

```diff
@@ -117,14 +24,35 @@ The provided object is first serialized, and then the byte array is used to gene
 
 2. Compute the estimate of the set so far `estimator.estimate()`.
 
-### Examples
+Please see the [cardinality estimator Javadoc](http://docs.hazelcast.org/docs/latest/javadoc/com/hazelcast/cardinality/CardinalityEstimator.html) for more information on its API.
+
+The following is an example code.
 
 ```
-HazelcastInstance hazelcast = Hazelcast.newHazelcastInstance();
-CardinalityEstimator uniqueVisitorsEstimator = instance.getCardinalityEstimator("visitors");
-for (String ip : _visitorIpsBuffer_) {
-  uniqueVisitorsEstimator.add(ip);
+public class Member {
```

Comment:
The snippet consists of a signle method. The class and method definitions should not be here. Leave only the important piece of logic here, please.